apps/ejabberd/include/ejabberd_odbc.hrl

```diff
@@ -0,0 +1 @@
+-define(QUERY_TIMEOUT, 5000).
```

Comment:
Do we need this macro here? Maybe there is other `.hrl` already included by most of the modules needing this one?